lms/templates/footer.html

```diff
@@ -83,3 +83,19 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+% if settings.FEATURES.get('ENABLE_COOKIE_CONSENT', False):
+<!-- Cookie Consent -->
```

Comment:
I'd prefer that this logic all live in a single file called cookie-consent.html, and that it be included in the same way as the segment integration above.

i.e.

    <%include file="widgets/cookie-consent.html" />